### PR TITLE
LPS-153319 Add set/update extended properties validation tests and randomize others

### PR DESCRIPTION
**Pull request by ccorreagg**
Related PR: https://github.com/brianchandotcom/liferay-portal/pull/119985

This PR contains unit tests for validating the Default Property Validator implemented for the Set/Update extended properties functionality for the APIs.

Furthermore, the tests of the `EntityExtensionHandler` class have been randomized.

### CODE REVIEW

**Comment by ccorreagg** [comment]
ci:test:relevant

File: portal-test/src/com/liferay/portal/kernel/test/util/RandomTestUtil.java [hunk 1]
```diff
@@ -78,6 +78,19 @@ public static <T extends Enum<?>> T randomEnum(Class<T> clazz) {
 		return enumConstants[_random.nextInt(enumConstants.length)];
 	}
 
+	public static float randomFloat() {
```
Comment (javierdearcos) [inline portal-test/src/com/liferay/portal/kernel/test/util/RandomTestUtil.java:81]: 👏 👏 👏

**Comment by ccorreagg** [comment]
ci:forward

**Comment by ccorreagg** [comment]
ci:forward

**Pull request closed without merging**